### ISSUE TITLE
Problems with checkPress

### ISSUE THREAD

**Issue by dmc81** [davidepalladino/Button#3]
checkPress does not appear to function as stated in the wiki "With this method you can get the actual press, both for short and long. If the pressure is continuous, you will not remain in the method". 

Running the example shortPressPullup.ino and pressing the button results  in many prints of "PRESSED!:", even if the button is only pressed for a short time.
Am I missing something here?
Regards
David.C.

**Comment by davidepalladino** [comment]
This problem I've solved during the development of [Air Analyzer](https://github.com/davidepalladino/Air-Analyzer) project. With this your issue I decided to check the new version too, especially for `shortPressPullup.ino` example. After an inspection of method code, the problem might be that the electric current to input pin is not stable for some microseconds. With small code, like the example, the problem is highlighted. In my case, and especially with the new version, this problem was not revealed.
Anyway, this evening (I hope) I will put the new version of this library.

Thank you for reporting. I'm so happy for your utilization.
